quil-py/pyproject.toml

```diff
@@ -27,7 +27,7 @@ sdist-include = ["README.md"]
 dev = [
     "ruff>=0.3.7",
     "maturin>=1.2.3",
-    "mypy>=1.13.0",
```

Comment:
Is this necessary? Is there a regression?

Reply:
Not sure why this happened, I think `uv` made this change on my behalf. Reverted.